src/Aquifer.API/Endpoints/TranslationPairs/Post/Endpoint.cs

```diff
@@ -0,0 +1,51 @@
+using Aquifer.API.Common;
+using Aquifer.API.Services;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.TranslationPairs.Post;
+
+public class Endpoint(AquiferDbContext dbContext, IUserService userService) : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Post("/translation-pairs");
+        Permissions(PermissionName.SetTranslationPair);
+    }
+
+    public override async Task HandleAsync(Request request, CancellationToken ct)
+    {
+        var checkForExistingTranslationPair =
```

Comment:
Similar comment as above, this could be an `AnyAsync` instead since we're just checking boolean status.